src/db/__init__.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+SQLALCHEMY_DATABASE_URL = "sqlite:///./linktious.db"
```

Comment:
Let's support it from env variables and this will be default url

Reply:
done